pkgs/top-level/all-packages.nix

```diff
@@ -9312,6 +9312,20 @@ in
     ruby_2_5
     ruby_2_6;
 
+  rubyMinimal = ruby.override {
+    useRailsExpress = false;
```

Comment:
Why would you strip this one?

Reply:
We can enable it by default, I have no idea about what any of these options do to be honest :sweat_smile:.

Reply:
railsExpress is a patchset on top of ruby that was developed by some rails developer to make things run faster. It doesn't affect the closure size but creates a non-vanilla version of ruby. Might as well use the vanilla ruby for the minimal version.